lib/exabgp/bgp/message/update/attribute/community/extended/rt_record.py

```diff
@@ -0,0 +1,33 @@
+from exabgp.bgp.message.update.attribute.community.extended import ExtendedCommunity
+from exabgp.bgp.message.update.attribute.community.extended import rt
+
+import logging
```

Comment:
Ah yes, correct, the "import logging" and the line above are not needed.

Thanks!